### PR TITLE
Update Messages.json

### PR DESCRIPTION
**Pull request by Nunting**
DONE.
More literal translation is needed, but it's not urgent right now. (Tired)

### CODE REVIEW

File: ko/Messages.json [hunk 1]
```diff
@@ -201,50 +224,158 @@
...
+  },
   },
   "lobby": {
     "SERVER_NAME": "[yellow]⚡ [red]PvP [yellow]⚡[red] 경쟁전",
-    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#87ceeb]시즌 VIII",
+    "SERVER_NAME_2V2": "[yellow]⚡ [red]Ranked [blue]2 vs 2 [yellow]⚡",
+    "SERVER_NAME_3V3": "[yellow]⚡ [red]Ranked [purple]3 vs 3 [yellow]⚡",
+    "SERVER_NAME_HEX": "[yellow]⚡ [red]Ranked [green]H E X [yellow]⚡",
+    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#ff00ff]시즌 VIII",
+    "SERVER_DESCRIPTION_2V2": "[#ff00ff]Season VII",
```
Comment (LeoSko) [inline ko/Messages.json]: But it's season VIII already. :)

File: ko/Messages.json [hunk 2]
```diff
@@ -201,50 +224,158 @@
...
+    "SERVER_DESCRIPTION_3V3": "[#ff00ff]Season VII",
+    "SERVER_DESCRIPTION_HEX": "[green]new quality level of hex\n[#ff00ff]Season VII",
```
Comment (LeoSko) [inline ko/Messages.json]: Can you please translate this? Most probably this won't change.

File: ko/Messages.json [hunk 3]
```diff
@@ -260,16 +391,68 @@
...
+    "START_MATCH_PLAYER_ENTRY_FORMAT": "[white]@[white] [yellow]@[white]",
+    "START_MATCH_TEAM_ENTRY_FORMAT": "[#@]@[white]: @",
+    "SURRENDER_USAGE": "[yellow]/surrender[white] 를 입력하여 항복",
+    "START_FORMAT": "[green]Map [yellow]@\n@\n@",
+    "START_FORMAT_UNRANKED": "[green]Map [yellow]@\nUnranked match\n@\n@",
     "WIN_FORMAT": "승리",
     "LOSE_FORMAT": "패배",
     "TIE_FORMAT": "무승부",
+    "FINISH_FORMAT": "#[yellow]@[] of [yellow]@[]",
+    "RATING_CALCULATION_NEXT_TEAM_ELIMINATION_WARNING": "[red]Your rating change will be calculated when some next team will be eliminated",
```
Comment (LeoSko) [inline ko/Messages.json]: Translate or exclude.

File: ko/Messages.json [hunk 4]
```diff
@@ -260,16 +391,68 @@
...
+    "DUEL_TARGET_LEFT": "[yellow]당신의 대결 상대 @가 로비를 떠났습니다"
   },
   "room": {
     "MATCH_END_FORMAT": "[@]@![white]@\n이유: @\n\n[yellow]ranked.ddns.net[white]에서 경기 리플레이를 볼 수 있습니다.\n경기 ID: [yellow]@",
     "MATCH_END_FORMAT_SPECTATOR": "[gray]경기 끝![white]@\n이유: @\n\n[yellow]ranked.ddns.net[white]에서 경기 리플레이를 볼 수 있습니다.\n경기 ID: [yellow]@",
+    "START_MATCH_1V1": "[white]@[white] vs @[white]\n승/패 [green]@[]/[red]@",
+    "START_MATCH_PLAYER_ENTRY_FORMAT": "[white]@[white] [yellow]@[white]",
+    "START_MATCH_TEAM_ENTRY_FORMAT": "[#@]@[white]: @",
+    "SURRENDER_USAGE": "[yellow]/surrender[white] 를 입력하여 항복",
+    "START_FORMAT": "[green]Map [yellow]@\n@\n@",
```
Comment (LeoSko) [inline ko/Messages.json]: Translate or exclude.

File: ko/Messages.json [hunk 5]
```diff
@@ -260,16 +391,68 @@
...
+    "START_FORMAT_UNRANKED": "[green]Map [yellow]@\nUnranked match\n@\n@",
```
Comment (LeoSko) [inline ko/Messages.json]: Translate or exclude.

File: ko/Messages.json [hunk 6]
```diff
@@ -260,16 +391,68 @@
...
     "WIN_FORMAT": "승리",
     "LOSE_FORMAT": "패배",
     "TIE_FORMAT": "무승부",
+    "FINISH_FORMAT": "#[yellow]@[] of [yellow]@[]",
+    "RATING_CALCULATION_NEXT_TEAM_ELIMINATION_WARNING": "[red]Your rating change will be calculated when some next team will be eliminated",
+    "RATING_CHANGE": "Your rating: [orange]@->@[white] @",
```
Comment (LeoSko) [inline ko/Messages.json]: Translate or exclude.

File: ko/Messages.json [hunk 7]
```diff
@@ -278,10 +461,65 @@
...
     "HARD_SHUTDOWN_ROOM": "점검으로 인해 경기가 재시작되는 중입니다. 등급 등급은 변경되지 않았습니다.",
     "TIEBREAKER_DAMAGE": "[white]각 코어가 [@]@[white]만큼 피해 입음.",
-    "TIEBREAKER_WARNING": "[red]1분[white]후 타이\n브레이커가 시작됩니다."
+    "TIEBREAKER_WARNING": "[red]1분[white]후 \n타이브레이커가 시작됩니다."
+    "TIEBREAKER_FORMAT": "@[yellow]\u26A1[red]@[yellow]\u26A1\n@",
+    "TIEBREAKER_USAGE": "[yellow]/ui - tiebreaker[]를 입력하여 이 경고를 작은 팝업으로 옮기기",
+    "TIEBREAKER_FORMAT_FULL": "@[yellow]\u26A1[red]@[yellow]\u26A1\n@\n\n[white]@",
+    "SPECTATOR_STATS_PLAYER_FORMAT": "@ [yellow]@[white] @",
+    "SPECTATOR_STATS_FORMAT": "@\n@\n@\n@",
+    "NOTHING_TO_WAIT": "[scarlet]Nothing to wait for",
```
Comment (LeoSko) [inline ko/Messages.json]: Translate or exclude.

File: ko/Messages.json [hunk 8]
```diff
@@ -278,10 +461,65 @@
...
+    "PLAYER_OFFERS_DRAW": "[green]플레이어 [white]@[green] ([#@]@[green])가 무승부를 제안합니다. 무승부를 받아들이지 않은 플레이어: [white]@",
+    "PLAYER_OFFERS_SURRENDER": "[green]동료 [white]@[green]가 항복을 제안합니다. 항복을 받아들이지 않은 팀원: [white]@",
+    "PLAYER_NOT_OFFERS_DRAW": "[green]플레이어 [white]@[green] ([#@]@[green]) 는 더이상 무승부를 원하지 않습니다.",
+    "PLAYER_NOT_OFFERS_SURRENDER": "[green]동료 [white]@[green] 는 더이상 항복을 원하지 않습니다.",
+    "DRAW_USAGE": "[yellow]/draw[] 를 입력하여 무승부를 받아들이기.",
+    "RECONNECTED_AFTER_ABANDONED": "경기를 포기한 후 다시 연결되었습니다. 계속 관전자로로 시청하거나 새로운 대결을 시작할 수 있습니다.",
+    "ADMIN_FORCED_TUTORIAL": "[green]서버가 처음이고 관리자가 튜토리얼을 강제 적용한 것 같습니다 .\n\n이 과정은 짧고, 그곳에서 당신은 간단한 것들을 배울 수 있습니다.\n\n10분 정도밖에 안걸립니다.",
+    "PLAYER_ABANDONED": "플레이어 @[white] from team [#@]@[]은 대결을 떠났고 다시는 돌아오지 않습니다."
+  },
+  "tutorial": {
```
Comment (LeoSko) [inline ko/Messages.json:488]: Translate or exclude this whole section.

File: ko/Messages.json [hunk 9]
```diff
@@ -278,10 +461,65 @@
...
+    "RECONNECTION_ADVANCED": "플레이어 @[white] ([#@]@[white])가 대기 시간을 @초 증가시켰습니다",
+    "NO_MORE_ADVANCEMENTS": "타이머 증가 횟수 소진",
+    "PLAYER_OFFERS_DRAW": "[green]플레이어 [white]@[green] ([#@]@[green])가 무승부를 제안합니다. 무승부를 받아들이지 않은 플레이어: [white]@",
+    "PLAYER_OFFERS_SURRENDER": "[green]동료 [white]@[green]가 항복을 제안합니다. 항복을 받아들이지 않은 팀원: [white]@",
+    "PLAYER_NOT_OFFERS_DRAW": "[green]플레이어 [white]@[green] ([#@]@[green]) 는 더이상 무승부를 원하지 않습니다.",
+    "PLAYER_NOT_OFFERS_SURRENDER": "[green]동료 [white]@[green] 는 더이상 항복을 원하지 않습니다.",
+    "DRAW_USAGE": "[yellow]/draw[] 를 입력하여 무승부를 받아들이기.",
+    "RECONNECTED_AFTER_ABANDONED": "경기를 포기한 후 다시 연결되었습니다. 계속 관전자로로 시청하거나 새로운 대결을 시작할 수 있습니다.",
+    "ADMIN_FORCED_TUTORIAL": "[green]서버가 처음이고 관리자가 튜토리얼을 강제 적용한 것 같습니다 .\n\n이 과정은 짧고, 그곳에서 당신은 간단한 것들을 배울 수 있습니다.\n\n10분 정도밖에 안걸립니다.",
+    "PLAYER_ABANDONED": "플레이어 @[white] from team [#@]@[]은 대결을 떠났고 다시는 돌아오지 않습니다."
```
Comment (LeoSko) [inline ko/Messages.json]: Is "from team" translatable?

File: ko/Messages.json [hunk 10]
```diff
@@ -201,50 +224,158 @@
...
+    "REPORT_USAGE": "당신은 [yellow]/report[] 대상의[yellow]플레이어 ID 번호[]를 입력하여 [yellow]/report[]할 수 있습니다.\n추신:영어로 작성해야 관리자가 알아들을 수 있습니다\n현재 접속자:\n@\n[white]예시: [yellow]/report 12345 griefer in lobby[]\n[white]예시 #2: [yellow]/report 23456 AFK in matches[]",
+    "YOU_WERE_REPORTED": "[red]당신은 @[red]에게 신고당했습니다. 사유:[white]@[red]. 관리자가 이를 확인할 것입니다.",
+    "REPORT_SAVED": "[green]당신의 신고한 대상 [white]@[green] ID[yellow]#@[green] 과 이유 : [white]@[green] 가 저장되었습니다.",
+    "PLAYER_ENTRY": "[white]ID#[yellow]@[]: @",
+    "NAME_NOT_ALLOWED_PREFIX": "당신의 이름\n[red]@[]\n은 허용되지 않았습니다. 다른 이름을 선택하세요.",
+    "CHAT_PREFIX_ALL": "[@]모두[]:",
+    "CHAT_PREFIX_SPECTATORS": "[@]관전자[]:",
+    "CHAT_PREFIX_TEAM_ONLY_FORMAT": "([#@]@[]):",
+    "CHAT_PREFIX_GROUP_ONLY_FORMAT": "([@]그룹원[]):",
+    "HISTORICAL_MATCH_ENTRY": "Match#@ @ [yellow]@\n[white]@\n@",
```
Comment (LeoSko) [inline ko/Messages.json]: Translate or exclude.

File: ko/Messages.json [hunk 11]
```diff
@@ -201,50 +224,158 @@
...
+    "BALANCE_BLOCKS": "[yellow]건설 블록[]\n20부터 시작해서, 건설 진행도에 비례해서 체력이 증가합니다   (기본 버전은 모든 과정에서 체력이 20).\n\n[yellow]보호막 프로젝터[]\n- 낙파 피해량을 33%까지 감소시키고, 보호막과 블록이 피해량을 나눠 받습니다\n- 건물에 피해를 입히는 대신 보호막이 덮인 건물 위로 발사되는 총알을 흡수합니다.\n- 아이템 폭발 대미지를 1/5로 감소시킵니다\n\n[yellow]코어[]\n- 기반 코어\uF868 [red]♥[]3500⇨3700\n- 조각 코어\uF869 [red]♥[]1100⇨1500\n\n모든 탄환은 5배의 넉백을 가집니다\n\n패럴랙스\uF801는 4배 더 강하게 끌어옵니다\n\n[yellow]상태이상[]\n[blue]젖음\uF7B2[] reduces explosiveness and flammability to 40%, increases power to 160%\n[cyan]Freezing\uF7B5[] reduces explosiveness and flammability to 10% and power to 60%",
```
Comment (LeoSko) [inline ko/Messages.json:313]: Here just a little thing left. :)

File: ko/Messages.json [hunk 12]
```diff
@@ -201,50 +224,158 @@
...
+    "BALANCE_STATS": "[yellow]능력치[]\n체력 변경점은 데미지 배수를 설정된 데미지 배수를 거치고 적용됩니다.  (로직 역호환 가능)\n\n대거\uF800: 재장전 14⇨13\n메이스\uF7FF: [red]♥[]500⇨650, 30⇨34, 재장전 14⇨11, [yellow]\uE86B[]4⇨7\n포트리스\uF7FE: [red]♥[]800⇨1000, 54⇨62\n노바\uF7FD: [red]♥[]120⇨160, 14⇨16\n퀘이사\uF7FB: [red]♥[]650⇨700\n크롤러\uF7FA: [red]♥[]180⇨220, 85⇨90, [green]⛷[]1⇨1.05\nAtrax\uF7F9: reload 10⇨9, [green]⛷[]0.5⇨0.8\n스피록트\uF7F8: 재장전 15⇨14, [green]⛷[]0.4⇨0.7\n아키드\uF7F8: [green]\u26F7[]0.5⇨0.6\n플레어\uF7F6: 9⇨6\n호라이즌\uF7F5: [red]♥[]340⇨270\n제니스\uF7F4: [red]♥[]700⇨580, 16⇨15\n안텀브라\uF7F3: +20%\n이클립스\uF7F2: [red]♥[]20000⇨21000, +25%\n모노\uF7F1: [red]♥[]100⇨70\n폴리\uF7F0: [red]♥[]400⇨280\n리소\uF7E7: [red]♥[]280⇨200, [yellow]\uE86B[]2⇨0, 건물 대상 피해  x1⇨x0.7, 발사체 관성이 사라짐\n민케\uF7ED: 건물 대상 피해  x1⇨x0.7\n브라이드\uF7EC: 80⇨70, 건물 대상 피해  x1⇨x0.7, 발사체 관성이 사라짐",
```
Comment (LeoSko) [inline ko/Messages.json:312]: I'll accept `BALANCE_` translations for now, but keep in mind they change frequently. Next season I'll have to remove translation to make it actual.

File: ko/Messages.json [hunk 13]
```diff
@@ -201,50 +224,158 @@
...
+    "ONLY_FOR_DONORS": "[red]당신은 명령을 실행할 수 있는 후원자가 아닙니다 @. [yellow]ranked.ddns.net[]에서 서버를 후원하는 것을 고려해 보세요",
+    "REPORT_USAGE": "당신은 [yellow]/report[] 대상의[yellow]플레이어 ID 번호[]를 입력하여 [yellow]/report[]할 수 있습니다.\n추신:영어로 작성해야 관리자가 알아들을 수 있습니다\n현재 접속자:\n@\n[white]예시: [yellow]/report 12345 griefer in lobby[]\n[white]예시 #2: [yellow]/report 23456 AFK in matches[]",
+    "YOU_WERE_REPORTED": "[red]당신은 @[red]에게 신고당했습니다. 사유:[white]@[red]. 관리자가 이를 확인할 것입니다.",
+    "REPORT_SAVED": "[green]당신의 신고한 대상 [white]@[green] ID[yellow]#@[green] 과 이유 : [white]@[green] 가 저장되었습니다.",
+    "PLAYER_ENTRY": "[white]ID#[yellow]@[]: @",
+    "NAME_NOT_ALLOWED_PREFIX": "당신의 이름\n[red]@[]\n은 허용되지 않았습니다. 다른 이름을 선택하세요.",
+    "CHAT_PREFIX_ALL": "[@]모두[]:",
+    "CHAT_PREFIX_SPECTATORS": "[@]관전자[]:",
+    "CHAT_PREFIX_TEAM_ONLY_FORMAT": "([#@]@[]):",
```
Comment (LeoSko) [inline ko/Messages.json:322]: Do not translate formats, which in fact don't have anything to translate.

File: ko/Messages.json [hunk 14]
```diff
@@ -201,50 +224,158 @@
...
+    "ONBOARDING_TAPPING_ON_SPECTATE_IN_LOBBY": "당신은 승격전을 [GREEN]관전[]하기 위해 이것을 [yellow]눌렀습니다[]. 이 기능 및 다른 많은 UI 기능은 명령어[yellow]/ui[]를 통해 활성화할 수 있습니다.\n\n'눌러서 관전하기'는 [yellow]/ui - tap_spectate[]를 통해 비활성화할 수 있습니다."
+  },
   },
   "lobby": {
     "SERVER_NAME": "[yellow]⚡ [red]PvP [yellow]⚡[red] 경쟁전",
-    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#87ceeb]시즌 VIII",
+    "SERVER_NAME_2V2": "[yellow]⚡ [red]Ranked [blue]2 vs 2 [yellow]⚡",
+    "SERVER_NAME_3V3": "[yellow]⚡ [red]Ranked [purple]3 vs 3 [yellow]⚡",
+    "SERVER_NAME_HEX": "[yellow]⚡ [red]Ranked [green]H E X [yellow]⚡",
+    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#ff00ff]시즌 VIII",
```
Comment (LeoSko) [inline ko/Messages.json]: Wrong color of the season. Use `[#87ceeb]`

**Review by LeoSko — CHANGES_REQUESTED** [review]
That's a great contribution! Please take a bit more time on highlighted issues and I'll deploy within an hour.

File: ko/Messages.json (hunk 11 above)
Comment (Nunting) [reply ko/Messages.json:313]: I'm not sure what that part means

**Pull request closed without merging**